code/game/objects/structures/crates_lockers/crates.dm

```diff
@@ -130,6 +131,141 @@
 				A.ex_act(severity)
 		qdel(src)
 
+
+
+

```

Comment:
Instead of blank lines, try code headers in block comment style.

Elaborates more and takes up a little less space than the amount of empty lines you have going on.

This much space (and more below) is unnecessary.